src/sage/graphs/graph_decompositions/modular_decomposition.pyx

```diff
@@ -123,6 +126,29 @@ cdef object _md_tree_node_to_md_tree_inner_rec(const md_tree_node *n,
 
 
 cdef object md_tree_node_to_md_tree(const md_tree_node *n, CGraphBackend Gb):
+    """
+    This function converts a modular decomposition tree (given as a pointer to a
+    md_tree_node) into an object of the Python class Node (which is then
+    converted into the correct output by :meth:`Graph.modular_decomposition`).
+
+    The graph backend is needed to convert the int stored into the md_tree_node
+    into the corresponding vertex of the Python graph.
+
+    This function deals with the case of an empty tree and then delegates the
+    actual conversion to :func:`_md_tree_node_to_md_tree_inner_rec`.
+
+    TESTS
```

Comment:
`TESTS:`

Reply:
done in [5cc82cc](https://github.com/sagemath/sage/pull/39038/commits/5cc82ccc4681b38f2fb6cbf725208c409da66ac5)